ui/pages/confirmations/components/send/amount/amount.tsx

```diff
@@ -128,10 +128,27 @@ export const Amount = ({
   ]);
 
   const balanceDisplayValue = useMemo(() => {
-    return fiatMode
-      ? `${getFiatDisplayValue(String(balance))} ${t('available')}`
-      : `${balance} ${asset?.symbol} ${t('available')}`;
-  }, [fiatMode, getFiatDisplayValue, balance, asset?.symbol, t]);
+    if (fiatMode) {
+      return `${getFiatDisplayValue(String(balance))} ${t('available')}`;
+    }
+
+    // For ERC1155 tokens, use name or just show balance without symbol
+    if (asset?.standard === ERC1155) {
+      const displayName = asset?.name ? ` ${asset.name}` : '';
+      return `${balance}${displayName} ${t('available')}`;
```

Comment:
<img width="284" height="123" alt="Image" src="https://github.com/user-attachments/assets/4b880418-2ec8-4913-b99c-4d5a4642b16a" />

Clearly there is a white space in the screenshot but not in the code. Should we put a white space here or I miss something?

```suggestion
      return `${balance} ${displayName} ${t('available')}`;
```